### PR TITLE
Add windows path support

### LINKED ISSUE #69: Add support for windows paths in standard logs parsing
Include windows paths in parsing regex

### PR DESCRIPTION
**Pull request by tonyredondo**
Closes #69 

This PR modifies the `regex` in the logging instrumentation to add support to windows paths.

image

### CODE REVIEW

File: instrumentation/testing/logging.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ type stdIO struct {
 	sync      *sync.WaitGroup
 }
 
-var LOG_REGEX_TEMPLATE = `^%s(?:(?P<date>\d{4}\/\d{1,2}\/\d{1,2}) )?(?:(?P<time>\d{1,2}:\d{1,2}:\d{1,2}(?:.\d{1,6})?) )?(?:(?:(?P<file>[\w\-. /]+):(?P<line>\d+)): )?(.*)\n?$`
+var LOG_REGEX_TEMPLATE = `^%s(?:(?P<date>\d{4}\/\d{1,2}\/\d{1,2}) )?(?:(?P<time>\d{1,2}:\d{1,2}:\d{1,2}(?:.\d{1,6})?) )?(?:(?:(?P<file>[\w\-. \/\\:]+):(?P<line>\d+)): )?(.*)\n?$`
```
Comment (fermayo) [inline instrumentation/testing/logging.go]:
```suggestion
var LOG_REGEX_TEMPLATE = `^%s(?:(?P<date>\d{4}\/\d{1,2}\/\d{1,2}) )?(?:(?P<time>\d{1,2}:\d{1,2}:\d{1,2}(?:.\d{1,6})?) )?(?:(?:(?P<file>[\w\-. /\\:]+):(?P<line>\d+)): )?(.*)\n?$`
```

`/` does not need escaping, does it?

Comment (tonyredondo) [reply instrumentation/testing/logging.go]: You're right, not needed, I've added it because regex101.com was complaining. Commiting your suggestion...

**Comment by fermayo** [comment]
Now that we are at it, can we write a simple unit test for the regex? Like the examples I gave you once through slack

**Comment by tonyredondo** [comment]
> Now that we are at it, can we write a simple unit test for the regex? Like the examples I gave you once through slack

Done.

**Pull request merged**